lib/cmd-start.js

```diff
@@ -1,28 +1,35 @@
 var getPort = require('get-port')
 
-var bankai = require('../http')
+var isElectronProject = require('./is-electron-project')
 var http = require('./http-server')
+var bankai = require('../http')
 
 module.exports = start
 
 function start (entry, opts) {
-  var handler = bankai(entry, opts)
-  var state = handler.state // TODO: move all UI code into this file
-  var server = http.createServer(function (req, res) {
-    if (req.type === 'OPTIONS') return cors(req, res)
-    handler(req, res, function () {
-      res.statusCode = 404
-      return res.end('No route found for ' + req.url)
-    })
-  })
+  isElectronProject(__dirname, function (err, bool) {
```

Comment:
wouldn't this always end up checking `bankai`'s own `package.json`? 🤔

Reply:
oh lol, totally right!

Reply:
`process.cwd()` would be a best fit?

Reply:
fixed!

Reply:
yeah, `process.cwd()` or `path.dirname(entry)` are both reasonable I think